### ISSUE TITLE
[Bug]: Changing the colors in light mode

### ISSUE THREAD

**Issue by rohinish404** [FrancescoXX/free-Web3-resources#405]
### Description

The home page looks a mix of random colors. Should be changed to more minimilistic like white background and dark gray text.

### To Reproduce

1. Go to home page
2. Click on light mode
3. See error

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/FrancescoXX/4c-site/blob/main/docs/CODE_OF_CONDUCT.md)

**Comment by tobySolutions** [comment]
Are you still working on this @rohinish404!!

**Comment by rohinish404** [comment]
Yeah, actually i was busy with some stuff..will make a pr:)

**Comment by tobySolutions** [comment]
Awesome @rohinish404. Please, do take your time and goodluck!

**Comment by tobySolutions** [comment]
Any updates @rohinish404?

**Comment by rohinish404** [comment]
I think this issue can be closed @tobySolutions as it has been resolved. The minor errors left can be resolved in the further issues :)

**Comment by tobySolutions** [comment]
Thanks @rohinish404!! Issue resolved by #409
